src/clj/game/cards/ice.clj

```diff
@@ -1565,30 +1601,6 @@
                    :prompt "Choose an ICE to install from HQ"
                    :effect (req (corp-install state side target (zone->name (first (:server run))) {:ignore-all-cost true}))}]}
 
-   "Formicary"
```

Comment:
How did this end up here???

Reply:
haha I have no idea how some of these ended up where they did.